lib/myapp_web/live/product_live/form_component.ex

```diff
@@ -3,6 +3,15 @@ defmodule MyappWeb.ProductLive.FormComponent do
 
   alias Myapp.Products
 
+  @impl true
+  def mount(socket) do
+    {:ok, assign(socket, :refreshing, false)}
+  end
+
+  def update(%{refreshing: refreshing}, socket) do
+    {:ok, assign(socket, :refreshing, refreshing)}
+  end
```

Comment:
We also need a dedicated update clause, because when `send_update` happens, it only sends the assigns specified in `send_update`. This may be a bug, not sure.